spec/20-BINARY_FORMAT.md

```diff
@@ -1,3 +1,131 @@
 # Binary format
 
-TBD
\ No newline at end of file
+Binary format document describes how to encode each field - `traceparent` and
+`tracestate`. The binary format should be used to encode the values of these
+fields. This specification does not specify how these fields should be stored
+and send as a part of a binary payload. The basic implementation may serialize
+those as size of the field followed by the value.
+
+# `Traceparent` binary format
+
+The field `traceparent` encodes the version of the protocol and fields
+`trace-id`, `parent-id` and `recorded`. Each field starts with the one byte
+field identifier with the field value following immediately after it. Field
+identifiers are used as markers for additional verification of the value
+consistency and may be used in future for the versioning of the `traceparent`
+field.
+
+``` abnf
+traceparent     = version version_format  
+version         = 1BYTE                   ; version is 0 in the current spec
+version_format  = "{ 0x0 }" trace-id "{ 0x1 }" parent-id "{ 0x2 }" recorded
+trace-id        = 16BYTES
+parent-id       = 8BYTES
+trace-flags     = 1BYTE  ; only the least significant bit is used
```

Comment:
As a minor nit, should the `recorded` above be replaced with `trace-flags` to match this?